compute/cloud-client/src/main/java/compute/CreateWithLocalSsd.java

```diff
@@ -0,0 +1,197 @@
+
+    createWithLocalSsd(project, zone, instanceName);
+  }
+
+  // Create a new VM instance with Debian 10 operating system and SSD local disk.
+  public static void createWithLocalSsd(String project, String zone, String instanceName)
+      throws IOException, ExecutionException, InterruptedException, TimeoutException {
+
+    // Get the latest debian image.
+    Image newestDebian = getImageFromFamily("debian-cloud", "debian-10");
+
+    // Create the disks to be included in the instance.
+    List<AttachedDisk> disks = new ArrayList<>();
+    String diskType = String.format("zones/%s/diskTypes/pd-standard", zone);
+    disks.add(diskFromImage(diskType, 10, true, newestDebian.getSelfLink(), true));
+    disks.add(localSsdDisk(zone));
+
+    // Create the instance.
+    Instance instance = createInstance(project, zone, instanceName, disks);
+
+    if (instance != null) {
+      System.out.printf("Instance created with local SSD: %s", instance.getName());
+    }
+
+  }
+
+  // Retrieve the newest image that is part of a given family in a project.
+  // Args:
+  //    project: project ID or project number of the Cloud project you want to get image from.
+  //    family: name of the image family you want to get image from.
+  private static Image getImageFromFamily(String project, String family) throws IOException {
+    // Initialize client that will be used to send requests. This client only needs to be created
+    // once, and can be reused for multiple requests. After completing all of your requests, call
+    // the `imagesClient.close()` method on the client to safely
+    // clean up any remaining background resources.
+    try (ImagesClient imagesClient = ImagesClient.create()) {
+      // List of public operating system (OS) images: https://cloud.google.com/compute/docs/images/os-details
+      return imagesClient.getFromFamily(project, family);
+    }
+  }
+
+  // Create an AttachedDisk object to be used in VM instance creation. Uses an image as the
+  // source for the new disk.
+  //
+  // Args:
+  //    diskType: the type of disk you want to create. This value uses the following format:
+  //        "zones/{zone}/diskTypes/(pd-standard|pd-ssd|pd-balanced|pd-extreme)".
+  //        For example: "zones/us-west3-b/diskTypes/pd-ssd"
+  //
+  //    diskSizeGb: size of the new disk in gigabytes.
+  //
+  //    boot: boolean flag indicating whether this disk should be used as a
+  //    boot disk of an instance.
+  //
+  //    sourceImage: source image to use when creating this disk.
+  //    You must have read access to this disk. This can be one of the publicly available images
+  //    or an image from one of your projects.
+  //    This value uses the following format: "projects/{project_name}/global/images/{image_name}"
+  //
+  //    autoDelete: boolean flag indicating whether this disk should be deleted
+  //    with the VM that uses it.
+  private static AttachedDisk diskFromImage(String diskType, int diskSizeGb, boolean boot,
```

Comment:
`createDiskFromImage` or `getDiskFromImage` would be a little clearer IMO

Reply:
Replaced with createDiskFromImage()